testsuite/features/secondary/min_rhlike_openscap_audit.feature

```diff
@@ -13,19 +13,7 @@ Feature: OpenSCAP audit of Red Hat-like Salt minion
     Given I am authorized for the "Admin" section
 
   Scenario: Enable repositories for openSCAP on the Red Hat-like minion
-    Given I am on the Systems overview page of this "rhlike_minion"
-    When I follow "Software" in the content area
-    And I follow "Software Channels" in the content area
-    And I wait until I do not see "Loading..." text
-    And I check radio button "no-appstream-8-result-RHEL8-Pool for x86_64"
-    And I check "no-appstream-8-result-Custom Channel for Rocky 8 DVD"
-    And I wait until I do not see "Loading..." text
-    And I click on "Next"
-    Then I should see a "Confirm Software Channel Change" text
-    When I click on "Confirm"
-    Then I should see a "Changing the channels has been scheduled." text
-    And I wait until event "Subscribe channels scheduled by admin" is completed
-    When I enable repository "Rocky-BaseOS" on this "rhlike_minion"
+    When I enable the repositories "Rocky-BaseOS Rocky-AppStream" on this "rhlike_minion"
```

Comment:
I might be wrong but, how you can have these channels "Rocky-BaseOS Rocky-AppStream" available if we don't trigger a channel sync for those first?
Even if we kill them later, but I think at least, we need to trigger them to have them available. 🤔

Reply:
These are Sumaform injected repositories present by default when we deploy the environment - hence the lack of needed syncing

Reply:
Ah that make sense!